packages/react-composites/src/composites/common/TabHeader.tsx

```diff
@@ -109,11 +110,22 @@ export const PeopleAndChatHeader = (props: PeopleAndChatHeaderProps): JSX.Elemen
           </Stack>
         </FocusZone>
       </Stack.Item>
-      {/* Hidden icon to take the same space as the actual back button on the left. */}
-      <DefaultButton
-        styles={mobilePaneHiddenIconStyles}
-        onRenderIcon={() => <CallWithChatCompositeIcon iconName="ChevronLeft" />}
-      ></DefaultButton>
+      {props.onHeaderMenuClick ? (
+        <DefaultButton
+          ariaLabel={strings.peoplePaneMoreButtonAriaLabel}
+          styles={concatStyleSets(mobilePaneBackButtonStyles, {
+            icon: { display: 'flex', alignItems: 'center' }
+          })}
+          iconProps={{ iconName: 'PeoplePaneMoreButton' }}
+          onClick={props.onHeaderMenuClick}
+        />
+      ) : (
+        /* Hidden icon to take the same space as the actual back button on the left. */
+        <DefaultButton
```

Comment:
Do we need to do anything here to ensure that this cannot be navigated to using talk back? I thought this guy had a tabIndex of `-1` or something?

Reply:
Can we ensure this is not captured using keyboard navigation @mgamis-msft ?

Reply:
Good catch! I will test that out and make sure it does not.